pkg/controller/kabaneroplatform/kabaneroplatform_controller.go

```diff
@@ -96,6 +97,14 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 	if err != nil {
 		return err
 	}
+	
+	// Index Rolebindings by name
+	if err := mgr.GetFieldIndexer().IndexField(&rbacv1.RoleBinding{}, "metadata.name", func(rawObj runtime.Object) []string {
```

Comment:
I was curious if this is required, or if it's just a performance boost.  It's fine - I've just never used indexes before and was curious.

Reply:
It was necessary when I was making the attempt with the structured List(). It would err about metadata.name not being indexed. I did not retry without when I realized I would have to use Unstructured.